### PR TITLE
Display more of the platform detection information

### PR DESCRIPTION
**Pull request by webOS101**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
platform sampler didn't show all the info available

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Show :allthethings:

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: packages/sampler/stories/default/Platform.js [hunk 1]
```diff
@@ -3,32 +3,35 @@ import platform from '@enact/core/platform';
...
-				<br />
-				Version: {version}
-				<br />
-				Gesture support: {gesture ? 'true' : 'false'}
-				<br />
-				Touch support: {touch ? 'true' : 'false'}
-			</div>
+			<Scroller>
+				<h3>
+					Platform:
```
Comment (Djspaceg) [inline packages/sampler/stories/default/Platform.js]: Do these have to be on their own line, or can they just be in one line? `<h3>Platform:</h3>`

Comment (webOS101) [reply packages/sampler/stories/default/Platform.js]: I don't think it matters to me.  If you feel strongly, we can change it!

Comment (Djspaceg) [reply packages/sampler/stories/default/Platform.js]: Oh boy yes. It's like the tag exploded. Please change this.

Comment (webOS101) [reply packages/sampler/stories/default/Platform.js]: I did.

**Pull request closed without merging**